doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -348,8 +348,12 @@ Creating Hints
    are given, the hint is added to the `core` database.)
 
    Outside of sections, these commands support the :attr:`local`, :attr:`export`
-   and :attr:`global` attributes. :attr:`global` is the default.  Inside sections,
-   only the :attr:`local` attribute is supported because hints are local to sections.
+   and :attr:`global` attributes. :attr:`global` is the default. Inside sections,
+   some commands only support the :attr:`local` attribute. These are
+   :cmd:`Hint Immediate`, :cmd:`Hint Resolve`, :cmd:`Hint Constructors`,
+   :cmd:`Hint Unfold`, :cmd:`Hint Extern` and :cmd:`Hint Rewrite`. Regardless
+   of the supported locality attributes, :attr:`local` is the default for hint
+   commands inside sections.
```

Comment:
"Regardless of the supported locality attributes" is confusing, can you improve that?

Reply:
What do you propose?

Reply:
Hard for me to suggest wording since I don't understand the first part of the sentence.

Reply:
Do you mean that local is the default even if the syntax doesn't let you specify that??

Reply:
Some commands only support local, and some support local / export / global. In all cases, local is the default.

Reply:
What about replacing that sentence with "local is the default locality for all hint commands."? If it is regardless, then do you really need to mention it?

Reply:
Perhaps start a new paragraph with "Inside sections, local is the default.  Some commands only support ...".

Reply:
This would break the parallelism with the above paragraph. I can remove the "regardless" phrasing though, if you believe it makes things clearer.

Reply:
Yes, seems clear now.